src/Strategy/Node/Npm/PackageLock.hs

```diff
@@ -6,6 +6,7 @@ module Strategy.Node.Npm.PackageLock (
   PkgLockJson (..),
   PkgLockDependency (..),
   NpmResolved (..),
+  PkgLockPackage (..),
```

Comment:
[Minor] Let's add a unit test to signify that peerCycleDep is a known and covered case.

Reply:
I think this could be good, but if I wrote it it would probably be similar to the `race` thing I had in an earlier revision. I don't really like the idea of infinite recursion being the failure case because of its effect on CI. 

Do you think it's worth bringing back a test case like this? It seemed a bit less necessary now since we've removed the possibility by not recursing at all for peer dependencies.